jsapp/js/components/modalForms/formMedia.es6

```diff
@@ -237,7 +237,7 @@ class FormMedia extends React.Component {
             {!this.state.isUploadFilePending && (
               <Dropzone
                 onDrop={this.onFileDrop.bind(this)}
-                className='dropzone-settings'
+                className='dropzone-settings dropzone-form-media'
```

Comment:
Should we name it `kobo-dropzone` and `kobo-dropzone--form-media`?

Reply:
And maybe move all dropzone related styles to separate file in `js/components/common`?

Reply:
![image](https://user-images.githubusercontent.com/16762675/121556940-51f90c00-c9e2-11eb-92a0-c1af33a902d8.png)

there's a lot of modal specific changes, but I guess the one in `_kobo.modal.scss` is the most generic so I'll move that one :D

Reply:
it would be good to make a `dropzone` overhaul and make it a BEM element with `m` tags for each of these modals, but maybe we should do it in a different PR

Reply:
Yeah ultimately it would be nice to have a generic dropzone styles. I'm pondering about the same problem with our dropdowns right now in https://github.com/kobotoolbox/kpi/issues/3266